packages/block-editor/src/components/inspector-controls-tabs/use-inspector-controls-tabs.js

```diff
@@ -42,16 +36,14 @@ export default function useInspectorControlsTabs( blockName ) {
 		color: colorGroup,
 		default: defaultGroup,
 		dimensions: dimensionsGroup,
-		list: listGroup,
 		position: positionGroup,
 		typography: typographyGroup,
 	} = InspectorControlsGroups;
 
 	// List View Tab: If there are any fills for the list group add that tab.
 	const listViewDisabled = useIsListViewTabDisabled( blockName );
-	const listFills = useSlotFills( listGroup.Slot.__unstableName );
 
-	if ( ! listViewDisabled && !! listFills && listFills.length ) {
+	if ( ! listViewDisabled ) {
```

Comment:
This check was removed to allow the navigation block to set the list view tab as the default regardless of whether fills have been rendered to that block or not.

Reply:
This change ended up being reverted due to the possibility of ending up with an empty tab.